test/fs.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const metatests = require('metatests');
+const fs = require('fs');
+const { mkdirp } = require('..');
+const path = require('path');
+const testMkdirp = metatests.test('mkdir');
+const mkdirpTestDir = 'test/ex1/ex2';
+
+testMkdirp.beforeEach((test, cb) => {
+  fs.rmdir(mkdirpTestDir, err => {
+    if (err && err.code === 'ENOENT') {
+      cb();
+      return;
+    }
+    test.error(err);
+    fs.rmdir(path.dirname(mkdirpTestDir), err => {
+      if (err && err.code !== 'ENOENT') {
+        test.error(err);
+      }
+      cb();
+    });
+  });
+});
+
+testMkdirp.afterEach((test, cb) => {
+  fs.rmdir(mkdirpTestDir, err => {
+    if (err) test.error(err);
+    cb();
+    fs.rmdir(path.dirname(mkdirpTestDir), err => {
+      if (err) test.error(err);
```

Comment:
Same here and `cb()` call should be here.
```suggestion
      if (err && err.code !== 'ENOENT') test.error(err);
      cb();
```